Docs: Update contributing docs to replace missing image and descriptions (#658)
FlowFuse/node-red-dashboard: docs/contributing/index.md

## Project Structure

### `/nodes` 

> Contains the files each node withat define each of the Node-RED nodes that make up the Dashboard 2.0 node set. You can read more about writing nodes for Node-RED [here](https://nodered.org/docs/creating-nodes/first-node).

Reviewer: ```suggestion
Contains the files that define each of the Node-RED nodes that make up the Dashboard 2.0 node set. You can read more about writing nodes for Node-RED [here](https://nodered.org/docs/creating-nodes/first-node).
```

### `/ui`

> Contains our VueJS app that forms to core of Dashboard 2.0. Within this, `/ui/src/widgets`, you will find the respective `.vue` files for each of the Dashboard 2.0 widgets.

Reviewer: ```suggestion
Contains our VueJS app that forms the core of Dashboard 2.0. Inside `/ui/src/widgets`, you will find all a set of subdirectories each containing a `.vue` file. It is these files that define the appearance and functionality a users sees when viewing the Dashboard. 
```

### `/docs`

A [VitePress](https://vitepress.dev/) documentation site that is used to generate the documentation for Dashboard 2.0 (what you're reading now).

## Installing Locally

### Pre-requisites

- [GitHub Account](https://github.com/) - You will need a GitHub account in order to make a copy of the code and contribute any changes.

> - [Node.js](https://nodejs.org/en/download) - Node.js will also come package with the Node Package Manager (`npm`) which is used to install dependencies, and run the Dashboard (and Node-RED) locally.

Reviewer: ```suggestion
- [Node.js](https://nodejs.org/en/download) - Node.js will also come packaged with the Node Package Manager (`npm`) which is used to install dependencies, and run the Dashboard (and Node-RED) locally.
```
